sdk/test-utils/test-utils/src/tracing/azureTraceMatcher.ts

```diff
@@ -109,3 +152,19 @@ function getSpanGraph(traceId: string, instrumenter: MockInstrumenter): SpanGrap
     roots,
   };
 }
+
+function sameArrayMembers<T>(expected: T[], actual: T[]): boolean {
+  expected = expected.sort();
+  actual = actual.sort();
+  if (expected.length !== actual.length) {
+    return false;
+  }
+
+  for (const item of expected) {
```

Comment:
The sorting above makes me think that you expect the array members to be ordered the same but this does not test for that (like including the positions, etc) - maybe we don't need to sort them at all?

If we do, can we use `toSorted` yet? I don't recall!

Reply:
We can't use `toSorted` until Node 20+ https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/toSorted